src/arrayMethodJoin.js

```diff
@@ -5,7 +5,27 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let anwser = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === undefined || this[i] === null) {
+        anwser += '';
+      } else {
+        anwser += this[i];
+      }
```

Comment:
```suggestion
      if (this[i] !== undefined && this[i] !== null) {
        anwser += this[i];
      }
```
Можно упростить, зачем плюсовать пустую строку?